dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEnrollmentAnalyticsManager.java

```diff
@@ -203,8 +204,22 @@ private boolean addValueMetaInfo( Grid grid, SqlRowSet rowSet, String columnName
 
                 boolean isSet = rowSet.getObject( columnName ) != null;
 
-                grid.addGridValueMeta(
-                    new GridValueMeta( columnName, gridRowIndex, getGridValueStatus( isDefined, isSet ) ) );
+                Map<Integer, Map<String, Object>> rowContext = grid.getRowContext();
+
+                Map<String, Object> row = rowContext.get( gridRowIndex );
+
+                if ( row == null )
+                {
+                    row = new HashMap<>();
+                }
+
+                Map<String, GridValueStatus> colValueType = new HashMap<>();
+
+                colValueType.put( "type", getGridValueStatus( isDefined, isSet ) );
```

Comment:
Maybe "type" is too generic...?
What about something like "eventRepertionPresence", "eventRepertionValue", etc.?

Reply:
There is repeatableStageValueStatus